nb/api/tint.md

```diff
@@ -0,0 +1,34 @@
+### tint()
+Parameter|Default|Type
+---|---|---
+color||The name of the color to tint with.
+
+Sets the color of all leaf geometry in shape to the named color, replacing existing colors.
+
+Mixes the named color with all leaf geometry, merging with existing colors.
```

Comment:
This seems contradictory to me. Is it replacing the existing color or mixing with the existing color. It's mixing with it, right?

Reply:
Oops, yes.

Reply:
Yes, you're right -- that first sentence needs deleting.